Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1140,6 +1140,7 @@ public class BrowserViewController: UIViewController {
     // Priority: P3A - VPN - Default Browser - Rewards
     // TODO: Remove the dispatch after with a proper fix and fix calling present functions before super.viewDidAppear
     DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+      self.presentBottomBarCallout()
```

Comment:
I would say P3A should show first. What do you think?

Reply:
Changed https://github.com/brave/brave-ios/pull/6640/commits/1b9470aca9072e5e1bcdfd337345da622ab950d1